src/sdk/SpeechTranslationConfig.ts

```diff
@@ -147,24 +126,6 @@ export abstract class SpeechTranslationConfig extends SpeechConfig {
      * @param {string} value - The name of the voice.
      */
     public abstract set voiceName(value: string);
-
-    /**
-     * Sets a named property as value
-     * @member SpeechTranslationConfig.prototype.setProperty
-     * @function
-     * @public
-     * @param {string} name - The name of the property.
-     * @param {string} value - The value.
-     */
-    public abstract setProperty(name: string, value: string): void;
-
-    /**
-     * Dispose of associated resources.
-     * @member SpeechTranslationConfig.prototype.close
-     * @function
-     * @public
-     */
-    public abstract close(): void;
```

Comment:
Same comment as above about removing API members.

Reply:
SpeechTranslationConfig extends SpeechConfig which already declares/defines these properties and methods.
There is no change in the exposed API.